docker/circleci/Dockerfile

```diff
@@ -103,12 +87,6 @@ RUN mkdir -p $GOPATH/src/github.com/operator-framework \
     && rm -rf /root/.cache \
     && rm -rf /go/pkg/dep/sources/
 
-
-
-# END IMAGE CUSTOMIZATIONS
-
-#USER circleci
-
-WORKDIR cassandra-k8s-operator
+WORKDIR /go/cassandra-k8s-operator
```

Comment:
Why adding /go there ?

Reply:
`Follow best practices`

Reply:
There is no need since we used go modules

Reply:
Not sure we're on the same page. What do you mean ? I'm just saying that it's a best practice to use absolute paths as workdir values, see https://docs.docker.com/v17.09/engine/userguide/eng-image/dockerfile_best-practices/

Reply:
Just why adding a /go in the path ?

Reply:
Read the part on workdir, it must be absolute. And this is where it goes when we don’t specify it. So I’m not changing anything, just applying best practices